README.md

```diff
@@ -72,44 +70,97 @@ how to change monitor address ?
 ## Worker
 
 ```go
-// Limit number of concurrent jobs execution.
-// Use worker.Unlimited (0) if you want no limitation.
-w := worker.New(worker.OneByOne)
-w.ErrHandler = func(e error) {
-	log.Println(e)
+import (
+	"log"
+	"strings"
+	"os"
+	"time"
+	"github.com/appscode/g2/worker"
+	"github.com/mikespook/golib/signal"
+)
+
+func ToUpper(job worker.Job) ([]byte, error) {
+	log.Printf("ToUpper: Data=[%s]\n", job.Data())
+	data := []byte(strings.ToUpper(string(job.Data())))
+	time.Sleep(10 * time.Second)
+	return data, nil
 }
-w.AddServer("127.0.0.1:4730")
-// Use worker.Unlimited (0) if you want no timeout
-w.AddFunc("ToUpper", ToUpper, worker.Unlimited)
-// This will give a timeout of 5 seconds
-w.AddFunc("ToUpperTimeOut5", ToUpper, 5)
-
-if err := w.Ready(); err != nil {
-	log.Fatal(err)
-	return
+
+func main(){
+
+	log.Println("Worker starting...")
+	defer log.Println("Worker shutdown")
+
+	w := worker.New(worker.Unlimited)
+	defer w.Close()
+
+	//error handling
+	w.ErrorHandler = func(e error) {
+		log.Println(e)
+	}
+
+	w.AddServer("tcp4","127.0.0.1:4730")
+
+	// Use worker.Unlimited if you want no timeout
+	w.AddFunc("ToUpper", ToUpper, worker.Unlimited)
+	// This will give a timeout of 5 seconds
+	w.AddFunc("ToUpperTimeOut5", ToUpper, 5)
+
+	if err := w.Ready(); err != nil {
+		log.Fatal(err)
+		return
+	}
+
+	go w.Work()
+
+	//Ctrl-C to exit
+	signal.Bind(os.Interrupt, func() uint { return signal.BreakExit })
+	signal.Wait()
 }
-go w.Work()
 ```
 
 ## Client
 
 ```go
-// ...
-c, err := client.New("tcp4", "127.0.0.1:4730")
-// ... error handling
-defer c.Close()
-c.ErrorHandler = func(e error) {
-	log.Println(e)
-}
-echo := []byte("Hello\x00 world")
-echomsg, err := c.Echo(echo)
-// ... error handling
-log.Println(string(echomsg))
-jobHandler := func(resp *client.Response) {
-	log.Printf("%s", resp.Data)
+import (
```

Comment:
This is not example. Examples are under `/examples` directory. If you need to add more example or fix those fix it there.
This is just a code sample. Stick with the format as it is, just fix things that needs to be fixed.